package-lock.json

```diff
@@ -0,0 +1,8610 @@
+{
```

Comment:
Could you add this file in a separate commit please? I don't think it has something to do with React 15.6 update.

Reply:
this package-lock.json locks the new version of the fresh packages added (React, ReactDom, React-test-renderer)